### ISSUE TITLE
Bobby Ray shippments do not arrive at Drassen Airport

### ISSUE THREAD

**Issue by calapine** [ja2-stracciatella/mod-wildfire-maps#8]
The cargo crate in front of Pablo's desk stays always stays open and empty, the ordered goods do not appear.

- I know about bribing Pablo,  I did this severel times, no avail
- I killed Pablo, goods do not arrive under his succesor either. 
- Punching Pablo did not help either
- Total of 10x orders was performed to rule out a fluke, none arrived.
- Bobby Rays email confirms that the shipment was sent to Drassen.
- Pablo acknoleges that something arrived, he says "hope I didn't mislay it"
![image](https://user-images.githubusercontent.com/76205879/102601607-b23c7e80-4120-11eb-9653-8e7d5a469a4a.png)

Edit: I forgot to mention I am using Straciatella 
**ja2-stracciatella_0.17.1-20201215+838adad_win-msvc64-142.exe**
Wildfire maps for Stracciatella, version 0.2.2. https://github.com/ja2-stracciatella/mod-wildfire-maps/
Steamversion, english of  JA2 and Wildfire

**Comment by ja2-derek** [comment]
Thank you for the bug report.

A few quick questions - 
- Do you receive an email from Bobby Ray for every order placed?
- What is Drassen's town loyalty? 
- Can you also attach a save game?

**Comment by calapine** [comment]
Hello. Sorry for my late response. 

I have to apologize, the error was probably caused by me, I think I didn't copy the all the mod files. After re-installing it works.

However, now I have another problem:

Skyrider cannot be found

- Spoke with Waldo, he gave his speech. The quest appears in the history log.
- Normally NPC in Drassen give a hint towards his location. They do not in this game.
- I have checked all the 4 possible swamp sectors in which he can spawn. I did not see him, nor his house.
- I none of the sectors I received the "Squad found someone in this sector "pop-up

Save game is attached! (It's Dead-is-Dead game) [](url)

[SaveGame11.zip](https://github.com/ja2-stracciatella/mod-wildfire-maps/files/5762017/SaveGame11.zip)

**Edit:** I noticed that the mods' "strategic-map-npc-placements.json" does not contain an entry for skyrider. The default Stracciatella version of the file has him listed.. Could that be the issue?

**Comment by ja2-derek** [comment]
> **Edit:** I noticed that the mods' "strategic-map-npc-placements.json" does not contain an entry for skyrider. The default Stracciatella version of the file has him listed.. Could that be the issue?

Yes, you are right. The released version of this mod works only with JA2 Stracciatella 0.17.0. If you are using nightly version, you need to update the said JSON file. You can take reference from https://github.com/ja2-stracciatella/mod-wildfire-maps/pull/10/commits/9389385d05167fe8a1732d70934773b78dd36e75

**Comment by calapine** [comment]
Thanks! I will do that. 

1. Are there any other files to update before a playthrough? Or would you recommend not playing the mod with the currently nightly? (I had severe performance issues with 0.17.0, the nightly fixes that.)


2. I am not an expert, but there are any ways I could help? Stracciatella and the mods are great, would love to give something back.

**Comment by ja2-derek** [comment]
1. As far as I remember that's the only one to requiring a change. There are some others but those are cosmetics like sector description and loading screens.
2. Thank you for that! That would depend what your are comfortable with. You can keep playing and reporting any bugs (and tolerate while we are working on them) and that's already helping a lot. If you are good at coding you can help out on the open issues here and in the main JA2:S repo. If you are comfortable with JSON editing and modding you can also help to fix the data issues.

**Comment by ShishkinP** [comment]
Shipment arrival and Skyrider presence are both functional on the latest build.
